addon/converter/fixture-converter.js

```diff
@@ -39,7 +39,8 @@ export default class {
     if ('function' === typeof serializer.transformFor) {
       // Use serializer.transformFor for compatibility with ember-data-model-fragments,
       // for context see https://github.com/danielspaniel/ember-data-factory-guy/issues/182
-      return serializer.transformFor(type).serialize;
+      let fragmentType = serializer.transformFor(type);
+      return fragmentType.deserialize.bind(fragmentType);
```

Comment:
>Should this be return fragmentType.serialize.bind(fragmentType);?

Tried that and got "Cannot read property 'modelName' of undefined" on the last line of this snippet.

```
    /**
      Changes serializer fallbacks for fragments to use `serializer:-fragment`
      if registered, then uses the default serializer.
       @method serializerFor
      @private
      @param {String} modelName the record to serialize
      @return {DS.Serializer}
    */
    serializerFor: function serializerFor(modelOrClass) {
      var modelName;

      if (typeof modelOrClass === 'string') {
        modelName = modelOrClass;
      } else {
        modelName = modelOrClass.modelName;
      }
```